apps/ledger-live-mobile/src/newArch/features/ModularDrawer/screens/AccountSelection/index.tsx

```diff
@@ -48,19 +46,31 @@ const AccountSelectionContent = ({
   );
   const listRef = useRef<FlatList>(null);
   const { t } = useTranslation();
-  const { colors } = useTheme();
 
   const renderItem = useCallback(
-    ({ item }: { item: AccountUI }) => {
+    ({ item }: { item: RawDetailedAccount }) => {
+      const accountUI = {
```

Comment:
I think this caused a regression in the information available in the 'Select account' page:

| before | after |
| ---- | ---- |
| <img width="544" height="1036" alt="Image" src="https://github.com/user-attachments/assets/d6194549-1510-4f6e-9104-311dd6af5dbb" /> | <img width="544" height="1036" alt="Image" src="https://github.com/user-attachments/assets/931eaf12-4d34-49b3-b614-29d53618653a" /> |